index.html

```diff
@@ -2295,7 +2295,7 @@ <h2>Web Development</h2>
 						<li>Eliminate incorrect code, blocking events, form errors, security issues, and outdated code.</li>
 						<li>Design interfaces to be responsive, and adaptable to different devices and user preferences.</li>
 						<li>Include useful files that expected by browsers, search engines, and other services.</li>
-						<li>Use the latest version of tools and software and make use existing features where able.</li>
+						<li>Use the latest version of tools and software and use existing features where able.</li>
```

Comment:
"Use the latest version of tools_, libraries,_ and software and use ~~existing~~ _native_ features where able."